content/en/docs/setup/upgrade/notice/index.md

```diff
@@ -1,44 +1,66 @@
 ---
-title: 1.2 Upgrade Notice
-description: Important changes operators must understand before upgrading to Istio 1.2.
+title: 1.3 Upgrade Notice
+description: Important changes to consider when upgrading to Istio 1.3.
 weight: 5
 aliases:
     - /docs/setup/kubernetes/upgrade/notice/
 ---
 
 This page describes changes you need to be aware of when upgrading from
-Istio 1.1 to 1.2.  Here, we detail cases where we intentionally broke backwards
+Istio 1.2 to 1.3.  Here, we detail cases where we intentionally broke backwards
 compatibility.  We also mention cases where backwards compatibility was
 preserved but new behavior was introduced that would be surprising to someone
-familiar with the use and operation of Istio 1.1.
+familiar with the use and operation of Istio 1.2.
 
 For an overview of new features introduced with Istio 1.2, please refer
-to the [1.2 release notes](/about/notes/1.2/).
-
-## Installation and Upgrade
-
-{{< tip >}}
-The configuration model for Mixer has been simplified. Support for
-adapter-specific and template-specific Custom Resources has been
-removed by default in 1.2 and will be removed entirely in 1.3.
-Please move to the new configuration model.
-{{< /tip >}}
-
-Most Mixer CRDs were removed from the system to simplify the configuration
-model, improve performance of Mixer when used with Kubernetes, and improve
-reliability in a variety of Kubernetes environments.
-
-The following CRDs remain:
-
-| Custom Resource Definition name | Purpose |
-| --- | --- |
-| `adapter`| Specification of Istio extension declarations |
-| `attributemanifest` | Specification of Istio extension declarations |
-| `template` | Specification of Istio extension declarations |
-| `handler` | Specification of extension invocations |
-| `rule` | Specification of extension invocations |
-| `instance` | Specification of extension invocations |
-
-In the event you are using the removed mixer configuration schemas, set
-the following Helm flags during upgrade of the main Helm chart:
-`--set mixer.templates.useTemplateCRDs=true --set mixer.adapters.useAdapterCRDs=true`
+to the [1.3 release notes](/about/notes/1.3/).
+
+## Installation and upgrade
+
+We simplified the configuration model for Mixer and removed support for
+adapter-specific and template-specific Custom Resource Definitions (CRDs)
+entirely in 1.3. Please move to the new configuration model.
+
+We removed the Mixer CRDs from the system to simplify the configuration
+model, improve Mixer's performance in Kubernetes deployments, and improve
+reliability in various Kubernetes environments.
+
+Istio now captures all ports by default. If your services listen on `127.0.0.1`,
```

Comment:
These should be under a different heading than installation and upgrade. L28 - L42. Not sure how we want to section them, could be Traffic Management or give each its own section?